pages/linux/genfstab.md

```diff
@@ -0,0 +1,20 @@
+# genfstab
+
+> Generate output suitable for addition to an fstab file.
+> More information: <https://man.archlinux.org/man/extra/arch-install-scripts/genfstab.8>.
+
+- Generate the output based on LABEL:
```

Comment:
This is a little confusing&mdash;where does LABEL come from?

Reply:
The LABEL is a filesystem volume identifier, usually assigned (but not limited to) during the volume creation. More information: https://wiki.archlinux.org/index.php/Persistent_block_device_naming#by-label

Reply:
Then can you update either the description or the command, because it is not clear what "LABEL" has to do with the example command itself.

Maybe something like this?
```suggestion
- Generate the output based on a label:
```
and then change the example from `path/to/mount_point` to `path/to/label`?

I don't use Arch Linux, so feel free to correct me!